gotests_test.go

```diff
@@ -612,7 +614,14 @@ func mustReadFile(t *testing.T, filename string) string {
 	if err != nil {
 		t.Fatal(err)
 	}
-	return string(b)
+
+	// The newly generated test are formatted using the imports pkg.
+	// We need do the same to the golden files, otherwise we might end up with formatting mismatch
+	res, err := output.ProcessImports(filename, b)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return string(res)
```

Comment:
The function you're modifying here is called `mustReadFile`, and it doesn't have any documentation.

In the future, when someone is reading or writing code here, if they see `mustReadFile(...)` calls, they would be very surprised that it doesn't just read a file, but tries to modify it (and fails if it's not a valid .go file).

I would suggest leaving `mustReadFile` as is and making a higher-level helper that reads a Go file and formats it (or renaming `mustReadFile` to match its new behavior):

```Go
func mustReadAndFormatGoFile(t *testing.T, filename string) string {
	b, err := ioutil.ReadFile(filename)
	if err != nil {
		t.Fatal(err)
	}
	fmted, err := output.ProcessImports(filename, b)
	if err != nil {
		t.Fatal(err)
	}
	return string(fmted)
}
```

Although, in reality [`imports.Process`](https://godoc.org/golang.org/x/tools/imports#Process) can read the file itself if you provide a `nil` byte slice, so `mustReadAndFormatGoFile` can be even simpler:

```Go
func mustReadAndFormatGoFile(t *testing.T, filename string) string {
	fmted, err := output.ProcessImports(filename, nil)
	if err != nil {
		t.Fatal(err)
	}
	return string(fmted)
}
```

If `mustReadFile` isn't used anywhere else at all, you can remove it.